src/jaguar.toit

```diff
@@ -69,6 +70,9 @@ main device/Device endpoints/List:
     // exceptions that might occur from that to avoid blocking
     // the Jaguar functionality in case something is off.
     catch --trace: run-installed-containers
+    if disabled:
+      network-free.up
+      container-done.down
```

Comment:
There is something odd about this. You're waiting for one container here, but what happens if we're running 5 that all want the network to be disabled?

Reply:
as discussed in person: there is a bug here.
Will be fixed with the NetworkManager introduction.